pyorbital/tests/test_tlefile.py

```diff
@@ -99,10 +99,13 @@ def fake_local_tles_dir(tmp_path, monkeypatch):
     """Make a list of fake tle files in a directory."""
     file_path = tmp_path / 'tle-202211180230.txt'
     file_path.touch()
+    time.sleep(1)
```

Comment:
should this be a session fixture so that we don't have to way 3 seconds for every test?

Reply:
Yes, good point. Not that it will have a difference right now I guess as the fake tle files are only used in one test at the moment, but that may change in the future of course.